### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by andrewda** [SouthEugeneRoboticsTeam/vision#9]

**Comment by JacksonCoder** [comment]
What would the tests look like, exactly?

**Comment by SUPERCILEX** [comment]
@andrewda lol, what he said 😂

**Comment by andrewda** [comment]
Making sure `cv_utils.py` functions work as intended, and maybe abstracting out some more methods and testing those. I basically want to make sure it works on the low-level with certain images.

**Comment by andrewda** [comment]
Right now we test that the output is what we expect, which is good and all, but making sure the individual steps work would be even better so we know what breaks when it breaks.

**Comment by SUPERCILEX** [comment]
Oh, K. Yeah, that SGTM.

**Comment by JacksonCoder** [comment]
I think we should use the `unittest` module for testing. I've used it before, and it's very complete and surprisingly easy to use.

**Comment by andrewda** [comment]
Yep. The problem right now is that the way this program is organized unit tests won't really work. We'll have to refactor quite a bit to make it all testable.
